k8s/charts/openebs/README.md

```diff
@@ -66,48 +67,55 @@ helm del --purge <openebs-chart-name>
+| `image.pullPolicy`                      | Container pull policy                         | `IfNotPresent`                            |
+| `apiserver.image`                       | Image for API Server                          | `quay.io/openebs/m-apiserver`             |
+| `apiserver.imageTag`                    | Image Tag for API Server                      | `0.9.0`                                   |
+| `apiserver.replicas`                    | Number of API Server Replicas                 | `1`                                       |
+| `apiserver.sparse.enabled`              | Create Sparse Pool based on Sparsefile        | `false`                                   |
+| `provisioner.image`                     | Image for Provisioner                         | `quay.io/openebs/openebs-k8s-provisioner` |
+| `provisioner.imageTag`                  | Image Tag for Provisioner                     | `0.9.0`                                   |
+| `provisioner.replicas`                  | Number of Provisioner Replicas                | `1`                                       |
+| `localProvisioner.image`                | Image for localProvisioner                    | `quay.io/openebs/provisioner-localpv`     |
+| `localProvisioner.imageTag`             | Image Tag for localProvisioner                | `0.9.0`                                   |
+| `localProvisioner.replicas`             | Number of localProvisioner Replicas           | `1`                                       |
+| `localProvisioner.basePath`             | BasePath for hostPath volumes on Nodes        | `/var/openebs/local`                                       |
+| `webhook.image`                         | Image for admision server                     | `quay.io/openebs/admission-server`        |
+| `webhook.imageTag`                      | Image Tag for admission server                | `0.9.0`                                   |
+| `webhook.replicas`                      | Number of admission server Replicas           | `1`                                       |
+| `snapshotOperator.provisioner.image`    | Image for Snapshot Provisioner                | `quay.io/openebs/snapshot-provisioner`    |
+| `snapshotOperator.provisioner.imageTag` | Image Tag for Snapshot Provisioner            | `0.9.0`                                   |
+| `snapshotOperator.controller.image`     | Image for Snapshot Controller                 | `quay.io/openebs/snapshot-controller`     |
+| `snapshotOperator.controller.imageTag`  | Image Tag for Snapshot Controller             | `0.9.0`                                   |
+| `snapshotOperator.replicas`             | Number of Snapshot Operator Replicas          | `1`                                       |
+| `ndm.image`                             | Image for Node Disk Manager                   | `quay.io/openebs/openebs/node-disk-manager-amd64` |
```

Comment:
Why 2 openebs ? Only one is required.

quay.io/openebs/node-disk-manager-amd64

Reply:
good catch.. will update README

Reply:
done